### DISCUSSION TITLE
Changing Color When Dropdown Item Clicked

### DISCUSSION THREAD

**Discussion by nikeshm** [saadeghi/daisyui#859] [Q&A]
I'm working with TailwindCSS and daisyUI to build a dropdown menu. The code is below.

The problem is that when a dropdown menu item is clicked on (when it becomes active), it becomes purple, even though I'm using the utility class `bg-white` on the `<li>` element. How can I customize the color of the dropdown menu items on click? I'd really prefer a solution that is CSS-only, as I am using the CDN versions of Tailwind and daisyUI (and therefore have no `tailwind.config.js` to modify).

Here is a working reproduction of my code. I am using Laravel, and this is from one of my `.blade.php` components stored in `resources/views/partials`.

```html
<link href="https://cdn.jsdelivr.net/npm/daisyui@2.15.3/dist/full.css" rel="stylesheet" type="text/css" />
<link href="https://cdn.jsdelivr.net/npm/tailwindcss@2.2/dist/tailwind.min.css" rel="stylesheet" type="text/css" />

<nav class="flex w-screen">
  <div class="container flex-1 flex justify-center m-auto">
    <div class="dropdown dropdown-hover">
      <a class="text-2xl">ABOUT</a>
      <ul class="dropdown-content menu items-center">
        <li class="bg-white"><a>Item</a></li>
        <li class="bg-white"><a>Item</a></li>
      </ul>
    </div>
  </div>
</nav>

```
Here is an interactive version of the code above: [https://codepen.io/DataBoi/pen/RwQEqvw](https://codepen.io/DataBoi/pen/RwQEqvw)

**Comment by Mashiane** [comment]
Check,https://github.com/saadeghi/daisyui/discussions/1280
